Firestore/core/src/firebase/firestore/local/leveldb_persistence_apple.mm

```diff
@@ -46,7 +47,8 @@
   return Path::FromNSString(NSHomeDirectory()).AppendUtf8(dot_prefixed);
 
 #else
-#error "Don't know where to store documents on this platform."
+  return Status{Error::Internal,
```

Comment:
I don't think this change is right.

All known, supported configurations have been enumerated. Any other Apple platform should fail to compile because we haven't considered how to store data on it.

As it stands this change allows the build to succeed on an unknown platform, but then the default configuration (of persistence enabled) will always fail, which is not useful.

Reply:
Yes, corrected.